modules/updating-eus-to-eus-upgrade-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-console_{context}"]
+= EUS-to-EUS upgrade using the web console
+
+.Prerequisites
+
+* Have access to the web console as a user with `admin` privileges.
+
+.Procedure
+
+. Upgrade any OLM Operators to versions that are compatible with both versions you are updating to. You can find these steps on the link:https://docs.openshift.com/container-platform/4.11/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Preparing for an Operator update] page.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
++
+To view the status of all machine config pools, click *Compute* -> *MachineConfigPools* and review the contents of the *Update status* column.
++
+[NOTE]
+====
+If your machine config pools have an `Updating` status, please wait for this status to change to `Updated`. This process could take several minutes.
+====
+
+. Set your channel to `eus-4.12`. 
++
+To set your channel, click *Administration* -> *Cluster settings* -> *Channel*. You can edit your channel by clicking on the current hyperlinked channel.
```

Comment:
The nav menu is `Administration -> Cluster Settings -> Channel` , see screenshot
![Screenshot 2023-03-08 at 1 37 07 PM](https://user-images.githubusercontent.com/12692381/223628409-eeaa0eb6-f12b-4888-b60c-73a73b3ac0d1.png)

Reply:
fixed 👍

Reply:
same here, did you submit the changes?